storybook/preview.tsx

```diff
@@ -79,5 +43,61 @@ export const decorators = [
       </div>
     )
   },
-  withQueryProvider,
 ]
+
+const preview: Preview = {
+  parameters: {
+    backgrounds: {
+      default: "White",
+      values: backgrounds,
+    },
+    docs: {
+      source: {
+        excludeDecorators: true,
+      },
+    },
+    options: {
+      storySort: {
+        method: "alphabetical",
+        order: [
+          "Introduction",
+          "Systems",
+          [
+            "*",
+            "Tailwind",
+            [
+              "Overview",
+              "Getting Started",
+              "Configuration",
+              "Working with Tailwind",
+              "*",
+              "Utility Class References",
+              ["Overview", "*"],
+            ],
+          ],
+          "Components",
+          "Helpers",
+          "Design Tokens",
+          "Deprecated",
+          "AIO",
+        ],
+      },
+    },
+    // docs: {
```

Comment:
My PR will update this so dw about it

Reply:
Oh. I didn't see this comment until now, but I removed it haha.